kubevirtci

```diff
@@ -95,11 +95,19 @@ function kubevirtci::install() {
 	${_kubectl} wait -n capk-system --for=condition=Available=true deployment/capk-controller-manager --timeout=10m
 }
 
+function kubevirtci::generate_kubeconfig() {
+        make clusterkubevirtadm-linux
+        bin/clusterkubevirtadm-linux-amd64 apply credentials --namespace e2e-test
+        bin/clusterkubevirtadm-linux-amd64 get kubeconfig --namespace=e2e-test --output-kubeconfig=kubeconfig-e2e
+        sed -i -r 's/127.0.0.1:[0-9]+/192.168.66.101:6443/g' kubeconfig-e2e
```

Comment:
please clarify `192.168.66.101`

Reply:
The kubevirtci environment has special hook to access this cluster, it change the IP to localhost and use port forward in order to access into this special environment
This hook works when you access the cluster from the hosting machine
This PR creates different requirement, the kubeconfig also has to be used from the cluster itself, from capk manager container, in this case we can't use the localhost IP (because we are not running in the hosting machine)
Therefore I changed the IP to the control plane (master) node IP (which is always `192.168.66.101`), this IP also can be accessed from the capk manager container

Reply:
👍